mesonbuild/backend/backends.py

```diff
@@ -625,6 +625,8 @@ def write_test_serialisation(self, tests, datafile):
             for a in t.cmd_args:
                 if hasattr(a, 'held_object'):
                     a = a.held_object
+                if isinstance(a, build.Executable):
```

Comment:
I think this should be `build.BuildTarget`, because, f.ex., the argument might be a DLL that is loaded at runtime.

Reply:
Done, reckon that should work